scripts/spi_flash/fatfs_api.h

```diff
@@ -26,7 +26,7 @@ struct ff_file_info {
     uint16_t modified_date;
     uint16_t modified_time;
     uint8_t  attrs;
-    char     name[13];
+    char     name[255];
```

Comment:
This doesn't appear correct.  The struct should match the `FILINFO` struct in `ff.h`:

```c
typedef struct {
    FSIZE_t	fsize;			/* File size */
    WORD	fdate;			/* Modified date */
    WORD	ftime;			/* Modified time */
    BYTE	fattrib;		/* File attribute */
#if FF_USE_LFN
    TCHAR	altname[FF_SFN_BUF + 1];/* Altenative file name */
    TCHAR	fname[FF_LFN_BUF + 1];	/* Primary file name */
#else
    TCHAR	fname[12 + 1];	/* File name */
#endif
} FILINFO;
```

With LFN enabled, there should be an `altname[13]` field followed by `name[256]`.